src/js/utils/log.js

```diff
@@ -39,6 +31,19 @@ export const logByType = (type, args, stringify = !!IE_VERSION && IE_VERSION < 1
   // add console prefix after adding to history
   args.unshift('VIDEOJS:');
 
+  // If there's no console then don't try to output messages, but they will
+  // still be stored in `log.history`.
+  //
+  // Was setting these once outside of this function, but containing them
+  // in the function makes it easier to test cases where console doesn't exist
+  // when the module is executed.
+  const fn = window.console && window.console[type];
+
+  // Bail out if there's no console.
+  if (!fn) {
+    return;
+  }
```

Comment:
At this point, we've already stored the arguments in history; so, we can safely bail out if there's nothing to do. This is cheaper than potentially running `JSON.stringify` and calling a no-op.